in_toto/models/link.py

```diff
@@ -27,6 +27,8 @@
 FILENAME_FORMAT_SHORT = "{step_name}.link"
 UNFINISHED_FILENAME_FORMAT = ".{step_name}.{keyid:.8}.link-unfinished"
 UNFINISHED_FILENAME_FORMAT_GLOB = ".{step_name}.{pattern}.link-unfinished"
+UNFINISHED_FILENAME_FORMAT_GLOB_DIR = \
+  ".{metadata_directory}/.{step_name}.{pattern}.link-unfinished"
```

Comment:
I think the first `.` needs to be dropped?

Also, as we discussed, I think we need a better way to do it. For one thing, we're using `/` as the separator and we need to check how that'd work on a Windows machine. @lukpueh do you have any thoughts? `glob.glob` does accept the separator.

Reply:
For the window machine, my first thought is to add another format and check both formats when signing with gpg, but there should be a better method.